core/start-blueos-core

```diff
@@ -25,6 +25,13 @@ else
     echo "Error: Docket socket file does not exist!"
 fi
 
+# Generate hardware identiication uuid
+if [ -e "/etc/blueos"]; then
+    machineid-cli --key blueos --parts cpu-cores cpuid os-name driver-serial mac-address > /etc/blueos/hardware-uuid
```

Comment:
Doing so we are going to get in the case of 4% conflict.
Ref: https://forums.raspberrypi.com/viewtopic.php?t=190197

But since we are going to have `blueos_id`, we should be fine.